packages/compass/src/app/index.tsx

```diff
@@ -29,6 +29,15 @@ window.addEventListener('error', (event) => {
   );
 });
 
+window.addEventListener(
+  'unhandledrejection',
+  (event: PromiseRejectionEvent) => {
+    void ipcRenderer?.call('compass:rejection:fatal', {
+      reason: event.reason.toString(),
```

Comment:
Reason is whatever the promise rejected as. I suppose it _could_ be an error? Not sure about the right way to turn that into an error.  Maybe I _do_ need ensureError...

Reply:
Could just mirror the uncaught-exception case here